src/compiler-debugging.md

```diff
@@ -7,6 +7,21 @@ advice about specific parts of the compiler (e.g. the [Queries Debugging and
 Testing chapter](./incrcomp-debugging.html) or the [LLVM Debugging
 chapter](./backend/debugging.md)).
 
+## Configuring the compiler
+
+By default, rustc is built without most debug information. To enable debug info, 
+set `debug = true` in your config.toml. 
+
+Setting `debug = true` turns on many different debug options (e.g., `debug-assertions`, 
+`debug-logging`, etc.) which can be individually tweaked if you want to, but many people 
+simply set `debug = true`. Check out the comments in `config.toml.example` for more info.
+
+In some cases, just setting these options will not trigger a rebuild,
+so if you changed it and you already have a compiler built, you might
+want to call `x.py clean` to force a rebuild.
```

Comment:
Hmm, if this happens it's an x.py bug. Have you seen this happen or are you just moving the existing test?

Reply:
Just moving the existing text.

Reply:
Ok. @Mark-Simulacrum am I right in thinking that if x.py doesn't rebuild after setting debug = true that's a bug? I don't want to recommend clearing the whole cache if it's not necessary.

Reply:
Hm, yes, this does indeed sound like a bug. I am rather surprised though.

Reply:
@rylev Let's remove this text then until someone actually encounters this bug. It sounds like this hasn't happened in practice.

Reply:
Done!